editor/test/integration/engine/native_extensions_test.clj

```diff
@@ -48,17 +49,17 @@
      (let [workspace (test-util/setup-workspace! world "test/resources/extension_project")
```

Comment:
We should change this to use `setup-scratch-workspace!` instead so we don't get false positives from the test when running locally.